Adding files for package installation

brynickson opened the pull request:
Adds `environment.yml` and `setup.py` files for installation, as well as a `README` for instructions on how to install.

Fixes #14 and #12

brynickson commented:
Good catch @mengesser, it should be `-e` and the reason the setup failed is because I had commented out the `long_description`. Should work now.

brynickson commented:
We should also decide on a license type, which I have for now commented out of the `setup.py`. However, that is outside the scope of this PR, so I will open a ticket for now.

mengesser commented:
`polarTransform` is missing from `environment.yml`. I can add that and push it, but do we also want `jupyter` and `jwst` for the notebooks?

brynickson commented:
> `polarTransform` is missing from `environment.yml`. I can add that and push it, but do we also want `jupyter` and `jwst` for the notebooks?

@mengesser I have added `polarTransform` to the install files. Since `jwst` isn't used by the `AstroBkgInterp` package itself, I don't think we should include it in the default environment. As such, I've changed its reference in the quickstart notebook to "optional" and sectioned off that part of the notebook.
